gin/middleware.go

```diff
@@ -9,6 +9,18 @@ import (
 	"github.com/gin-gonic/gin"
 )
 
+// New returns a function that satisfies gin.HandlerFunc interface
+func New(engine *gin.Engine, notifier *gobrake.Notifier) func(c *gin.Context) {
+	return func(c *gin.Context) {
+		_, metric := gobrake.NewRouteMetric(context.TODO(), c.Request.Method, c.FullPath())
```

Comment:
Why `c.FullPath()` instead of `routeName`? We want to report routes, not paths.

Reply:
Ok, good find. Could you mark the old function as deprecated?

Reply:
ok